### ISSUE TITLE
Strategy guide

### ISSUE THREAD

**Issue by Sesu8642** [Sesu8642/FeudalTactics#51]
From an email I received: "A guidebook for cool strategies, sorta like an outline of strategies like divide and conquer etc". Could be realized as an optional "advanced" tutorial. Or maybe even a video.

**Comment by elchmartin** [comment]
Interesting Idea.
THough I think it is part of the fun to learn all of this.

**Comment by Sesu8642** [comment]
I agree. It could be locked until you win a game on very hard or something.
Maybe an "unofficial" video on Youtube would be better suited anyway - not linked and only to be found by those explicitly searching for this information. I will add the "help wanted" label in case anyone wants to create one.
